### ISSUE TITLE
[TS] Bit commands failing

### ISSUE THREAD

**Issue by Meemaw** [teambit/bit#1332]
## Expected Behavior

Running `bit status / bit tag` used to work but when from today I get error.

## Actual Behavior

Doesn't work.

## Error

```
error: bit failed to load bit.envs/compilers/typescript@0.0.5 with the following exception:
Cannot find module '@bit/bit.utils.object.group-by'.
Error: Cannot find module '@bit/bit.utils.object.group-by'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:548:15)
    at Function.Module._load (internal/modules/cjs/loader.js:475:25)
    at Module.require (internal/modules/cjs/loader.js:598:17)
    at require (internal/modules/cjs/helpers.js:11:18)
    at Object.<anonymous> (/Users/msnuderl/Workspace/Personal/meshwatch/common/.bit/components/compilers/typescript/bit.envs/0.0.5/dist/index.js:19:23)
    at Module._compile (internal/modules/cjs/loader.js:654:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
    at Module.load (internal/modules/cjs/loader.js:566:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:506:12)
    at Function.Module._load (internal/modules/cjs/loader.js:498:3)
## Specifications
```

  - Bit version: 13.0.4
  - Node version: v9.11.2
  - npm / yarn version: yarn: 1.12.3
  - Platform: OSx
  - Bit compiler (include version): Typescript 0.0.5

**Comment by GiladShoham** [comment]
@Meemaw 
please try to force an installing of the compiler by removing:
`.bit/components/*` or (if you use git in the project) `.git/bit/components/*`

**Comment by davidfirst** [comment]
This generally happens when moving the workspace to a different location, which breaks the symlink Bit generates.
With the newly released version (v14.1.0) it's possible to detect these issues by running `bit doctor`. it suggests a solution about how to fix it.
See https://github.com/teambit/bit/issues/1585 for more info.
